packages/oasis-actions/src/operations/aave/v2/adjustRiskUp.ts

```diff
@@ -128,10 +128,11 @@ export async function adjustRiskUp({
   ]
 
   const takeAFlashLoan = actions.common.takeAFlashLoan({
+    isDPMProxy: proxy.isDPMProxy,
+    asset: addresses.DAI,
     flashloanAmount: flashloan.amount,
-    borrower: addresses.operationExecutor,
     isProxyFlashloan: true,
-    isDPMProxy: proxy.isDPMProxy,
+    provider: new BigNumber(0),
```

Comment:
Would be nice to have an enum or some constant with the providers. Something like
```
cont FlashloanProviders = {
  DssFlash: ZERO,
  Balancer: ONE,
}
```
Right now these providers are like an enum in the contract sand correspond to numbers but at ay point of time it could be a `provider` configuration or could be replaced with a string.

Abstracting it like that you will change the value  in the definition only and  you should worry about where it's been used.

**Edit**: Actually I wrote that comment before I saw you actually defined such a constant.
1st. Why isn't it used everywhere?
2nd. What do you have to wrap it into a BigNumber?

If it needs to be a BigNumber, just use `ZERO` and `ONE` in the definition instead of wrapping it everywhere.